### ISSUE TITLE
Fails with exit code 127

### ISSUE THREAD

**Issue by otbe** [donaldpipowitch/workspace-release#1]
Hi,

I tried runnign your example but I get
```
20:21 $ yarn release
yarn run v1.7.0
$ workspace-version
env: node\r: No such file or directory
error Command failed with exit code 127.
```

I guess this is due to CRLF style (windows :D). If I change it to LF (via vscode) it works :)

Thanks!

**Comment by donaldpipowitch** [comment]
I always thought my `.gitattributes` with `* text=auto` would prevent these kind of errors 🤔

Should it be `* text=auto eol=lf` instead? (See https://stackoverflow.com/a/42135910)

**Comment by donaldpipowitch** [comment]
Should be fixed in version `0.4.4`.
